src/org/zaproxy/zap/extension/exportreport/ExportReportAPI.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+package org.zaproxy.zap.extension.exportreport;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.Constant;
+import org.zaproxy.zap.extension.api.ApiAction;
+import org.zaproxy.zap.extension.api.ApiException;
+import org.zaproxy.zap.extension.api.ApiException.Type;
+import org.zaproxy.zap.extension.api.ApiImplementor;
+import org.zaproxy.zap.extension.api.ApiResponse;
+import org.zaproxy.zap.extension.api.ApiResponseElement;
+import org.zaproxy.zap.extension.api.ApiResponseList;
+import org.zaproxy.zap.extension.api.ApiView;
+import org.zaproxy.zap.extension.exportreport.filechooser.FileList;
+import org.zaproxy.zap.extension.exportreport.filechooser.FileType;
+
+import net.sf.json.JSONObject;
+
+/**
+ * The API for export report.
+ *
+ * AUTHOR: GORAN SARENKAPA - JordanGS
+ * SPONSOR: RYERSON UNIVERSITY
+ *
+ */
+public class ExportReportAPI extends ApiImplementor {
+
+    private static final Logger logger = Logger.getLogger(ExportReportAPI.class);
```

Comment:
@thc202 for consitency it was `logger` since it was longer case in every other class after the initial review? Did you want me to change them all to `LOGGER`?